Public/Src/App/Bxl/Args.cs

```diff
@@ -784,6 +784,12 @@ public bool TryParse(string[] args, PathTable pathTable, out ICommandLineConfigu
                             opt =>
                             {
                                 var parsedOption = CommandLineUtilities.ParseEnumOption<SandboxKind>(opt);
+#if PLATFORM_OSX
+                                if (parsedOption == SandboxKind.MacOsEndpointSecurity && !OperatingSystemHelper.IsMacOSCatalinaOrHigher)

```

Comment:
>EndpointSecurity [](start = 69, length = 16)

No I think I know what ES stands for :) I always read Engineering System :P

Reply:
haha, yeah its EndpointSecurity - sorry 💃